src/cmd-remote-build-container

```diff
@@ -33,13 +34,22 @@ def push_container_image(repo, tag):
     cmd = ["podman", "--remote", "push", f"{repo}:{tag}"]
     # Long running command. Send output to stdout for logging
     runcmd(cmd, capture_output=False)
-    # Even though check_call only returns after the cmd completion,
-    # Quay seems to take more time to publish images in some occasions
-    runcmd(["sleep", "50"])
-    if is_tag_in_registry(repo, tag):
-        logger.info(f"Build and Push done successfully via tag: {tag}")
+    # Quay seems to take some time to publish images in some occasions.
+    # After the push let's wait for it to show up in the registry
+    # before moving on.
+    retryer = tenacity.Retrying(
```

Comment:
I am hopeful in the future we can drop more of these 3rd party python dependencies and use Go vendoring.